docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md

```diff
@@ -130,3 +130,25 @@ If everything is set up correctly, you can follow [How to send request in Data E
 Or if you're in the middle of the integration process for the Data Exchange API follow [Data Exchange API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/data-exchange-api-integration.html) to proceed with it.
 
 {% endinfo_block %}
+
+## Download Data Exchange API specification
+
+You can download the specification of for all available endpoints including the Data Exchange API endpoints in the OpenAPI format. To do so, click **Download API Specification** on the **Data Exchange API Configuration** page.
+
+[PASTE SCREENSHOT deapi_download_spec_button.png HERE]
+
+After you click the button, the specification will be downloaded to your computer.
+
+If you added new endpoints, documentation will be updated automatically with the new endpoints, but need wait 1 minute for it. Button **Download API Specification** will be disabled during this time and aviailable after 1 minute. You can see  message about it in the top of the page:
+
+[PASTE SCREENSHOT deapi_generation_in_progress.png HERE]
+
+***Also if for some reason file not exists, you will see the message about it***
+
+[PASTE SCREENSHOT deapi_spec_file_not_exists.png HERE]
+
+Please regenerate specification in this case manually or add new endpoints for Data Exchange API.
+
+```bash 
+ vendor/bin/glue api:generate:documentation
+```
```

Comment:
This is no longer the case, this functionality is removed. Please align with Dmitry Tsemma, and remove those lines and screenshot.